src/components/SignInButton/type.ts

```diff
@@ -1,3 +1,18 @@
+import { LoginScreenStylesType } from '../../screens/LoginScreen/style';
+import { IconDefinition } from '@fortawesome/fontawesome-common-types';
+
 export type SignInButtonPropsType = {
-  googleSignIn(): void;
+  signIn: SignInType;
+};
+type SignInType = {
+  signInMethod(): void;
+  title: string;
+  styleFont:
```

Comment:
try to use here just TextStyle and further the same, it have to work

Reply:
yes, it works